Source/WebCore/dom/Document.cpp

```diff
@@ -27,7 +27,7 @@
 
 #include "config.h"
 #include "Document.h"
-
+#include "DocumentInlines.h"
```

Comment:
This looks likely incorrect to add in this PR, but I'm not sure.

Reply:
Yea I was trying a few things last night. This PR was draft so I was planning to clean it up after I got a successful build first. But thanks for reviewing it anyway!